packages/bitcore-wallet-service/start-docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Run each service directly
+node ./ts_build/messagebroker/messagebroker.js &
+node ./ts_build/bcmonitor/bcmonitor.js &
+node ./ts_build/emailservice/emailservice.js &
+node ./ts_build/pushnotificationsservice/pushnotificationsservice.js &
+node ./ts_build/fiatrateservice/fiatrateservice.js &
+node ./ts_build/bws.js &
+
+echo "All services started"
+
+# Keep the script running
```

Comment:
Super hacky/lazy, open to something better

Reply:
Does tailing logs keep the container running instead of the forever loop?